### PR TITLE
Add me.ppvan.psequel

### PR DESCRIPTION
**Pull request by ppvan**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build me.ppvan.psequel

File: me.ppvan.psequel.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    "modules": [
+        {
+            "name": "psequel",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/ppvan/psequel",
+                    "tag": "v0.1.2",
+                    "commit": "2a8f412d22b87dd706c6e31d0fa248fdd94a5c94"
```
Comment (hfiguiere) [inline me.ppvan.psequel.json:42]: tag and commit don't match.

that's the build failure.

Comment (ppvan) [reply me.ppvan.psequel.json]: my bad, i updated the checksum

**Comment by ppvan** [comment]
bot, build me.ppvan.psequel

**Comment by ppvan** [comment]
@hfiguiere seems like the blueprint broken build chain. But i can't reproduce it in local (with gnome builder), i deleted every thing and reclone

**Comment by hfiguiere** [comment]
gnome builder is NOT `flatpak-builder` . so the comparison is irrelevant.

Did you build is locally with `flatpak-builder`? Is every source file needed in the archive?

**Comment by ppvan** [comment]
bot, build me.ppvan.psequel

**Comment by ppvan** [comment]
Why i don't see anything at [https://flathub.org/apps/me.ppvan.psequel](https://flathub.org/apps/me.ppvan.psequel) ?
The app do show up in my account

**Pull request closed without merging**